components/tikv_util/src/memory.rs

```diff
@@ -181,13 +181,15 @@ impl std::error::Error for MemoryQuotaExceeded {}
 
 impl_display_as_debug!(MemoryQuotaExceeded);
 
+#[derive(Debug)]
 pub struct MemoryQuota {
     // We suppose the memory quota should not exceed the upper bound of `isize`.
     // As we don't support 32bit(or lower) arch, this won't be a problem.
     in_use: AtomicIsize,
     capacity: AtomicUsize,
 }
 
+#[derive(Debug, Clone)]
```

Comment:
```suggestion
#[derive(Debug)]
```

Nit: Implementing Clone for OwnedAllocated might be error-prone as it could lead to unexpected free operations if cloned accidentally.

Reply:
Thanks for the reminding, I've mis-understanded the `OwneAllocated`, PTAL again